editor/src/components/canvas/__snapshots__/ui-jsx-canvas.spec.tsx.snap

```diff
@@ -4226,11 +4347,25 @@ exports[`UiJsxCanvas render class component is available from arbitrary block in
       width: 400px;
       height: 400px;
     \\"
+    data-paths=\\":utopia-storyboard-uid\\"
     data-uid=\\"utopia-storyboard-uid\\"
   >
-    <div data-uid=\\"zzz scene-aaa\\">
-      <div data-uid=\\"ccc aaa\\">Thing</div>
-      <div data-uid=\\"ccc bbb\\">Thing</div>
+    <div
+      data-uid=\\"zzz scene-aaa\\"
+      data-paths=\\"utopia-storyboard-uid/scene-aaa:zzz :utopia-storyboard-uid/scene-aaa\\"
+    >
+      <div
+        data-uid=\\"ccc aaa\\"
+        data-paths=\\"utopia-storyboard-uid/scene-aaa:zzz/aaa\\"
+      >
+        Thing
+      </div>
+      <div
+        data-uid=\\"ccc bbb\\"
+        data-paths=\\"utopia-storyboard-uid/scene-aaa:zzz/bbb\\"
+      >
```

Comment:
Looks like something has gone wrong here, because both of these elements should have 2 paths - both are missing a path ending in `:ccc`